conf/code-lists/holder-of-authorisation-codes.csv

```diff
@@ -1,55 +1,53 @@
-FZ
-GGA
-GVS
-IPO
-LVBI
-MOU
-NIRE
-OPO
-REM
-REP
-REX
-RORO
-RSS
-SAS
-SASP
-SDE
-SIVA
-SSE
-TEA
-TEAH
-TRD
-TST
-UKCS
+"Code","Description"
+"ACE","Authorised consignee for Union transit"
+"ACP","Authorised issuer to establish the proof of the customs status of Union goods"
+"ACR","Authorised consignor for Union transit"
+"ACT","Authorised consignee for TIR procedure"
+"AEOC","Authorised Economic Operator - Customs simplifications"
+"AEOF","Authorised Economic Operator - Customs simplifications/Security and safety"
+"AEOS","Authorised Economic Operator - Security and safety"
+"APEX","Approved Exporter"
+"AWB","Authorised Weigher of bananas"
+"BOI","Binding Origin Information"
+"BTI","Binding Tariff Information"
+"CCL","Authorisation for centralised clearance"
+"CGU","Comprehensive guarantee"
+"CSDR","Community System of Duty Reliefs"
+"CSE","Customs Supervised Exports"
+"CVA","Simplification of the determination of amounts being part of the customs value of goods"
+"CW1","Operation of storage facilities for the customs warehousing of goods in a public customs warehouse type I"
+"CW2","Operation of storage facilities for the customs warehousing of goods in a public customs warehouse type II"
+"CWP","Operation of storage facilities for the customs warehousing of goods in a private customs warehouse"
+"DEP","Designated Export Place"
+"DPO","Deferment of payment authorisation"
+"EIR","Authorisation for the use of customs declaration through an entry of data in the declarant’s records"
+"EORI","Economic Operator Registration and Identification"
+"EPSS","Excise Payment Security System"
+"ETD","Electronic transport document (authorised for use as a customs declaration)"
+"EUS","End use authorisation"
+"EXEE","Registered Consignee (Excise)"
+"EXOR","Registered Consignor (Excise)"
+"EXW","Excise Warehouse"
+"EXWH","Excise Warehouse - Hydrocarbon Oils"
+"FAS","Flexible Accounting System"
+"FZ","Free Zone (only for the use of the Isle of Man)"
+"GGA","General Guarantee Account"
+"GVS","General Valuation Statement"
+"IPO","Inward processing procedure authorisation"
+"LVBI","Low Value Bulking of Imports"
+"MOU","Memorandum of Understanding"
+"OPO","Outward processing procedure authorisation"
+"REM","Remission of the amounts of import or export duty"
+"REP","Repayment of the amounts of import or export duty"
+"REX","Registered Exporter"
+"RSS","Regular shipping services authorisation"
+"SAS","Authorisation for self-assessment"
+"SASP","Single Authorisation for Simplified Procedures"
+"SDE","Simplified declaration authorisation (SDP)"
+"SIVA","Simplified Import VAT Accounting"
+"SSE","Authorisation for the use of seals of a special type"
+"TEA","Temporary admission authorisation"
+"TEAH","Temporary Admission authorisation - Auction Houses (no guarantee for VAT required"
+"TRD","Authorisation to use transit declaration with a reduced dataset"
+"TST","Authorisation to operate storage facilities for the temporary storage of goods"
+"UKCS","UK Continental shelf"
```

Comment:
You have two codes less than before. Is it intentional?

Reply:
Yes - the two no-deal codes that have been removed from the tests.  TBH I did not check the actual before and after codes.  I just created the csv file from the contents of the Google doc i.e. assumed that doc was the source of truth.